libr/anal/fcn.c

```diff
@@ -1544,7 +1544,54 @@ R_API int r_anal_fcn_cc(RAnal *anal, RAnalFunction *fcn) {
 	return result;
 }
 
-R_API char *r_anal_fcn_to_string(RAnal *a, RAnalFunction *fs) {
+R_API char *r_anal_function_get_name(RAnalFunction *function) {
```

Comment:
`get_name` is not a good name for this. It would suggest something like just `return function->name;`. What about something like `get_signature`?

Reply:
I agree with this, let me change it